pandas/tests/series/methods/test_shift.py

```diff
@@ -344,3 +344,15 @@ def test_shift_preserve_freqstr(self, periods):
             index=pd.date_range("2016-1-1 02:00:00", periods=periods, freq="H"),
         )
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "input_value, expected_value",
+        [
+            (np.empty(shape=(0,)), pd.Series([], dtype="float64")),
+            (np.ones(shape=(2,)), pd.Series([np.nan, 1.0], dtype="float64")),
```

Comment:
Nit: You can move the `Series` call inside the test since it's shared here

Reply:
Thanks for the optimization, I changed it.